### DISCUSSION TITLE
Bash completion on parameter

### DISCUSSION THREAD

**Discussion by eeezyy** [knqyf263/pet#283] [Ideas]
When preparing some commands in pet, I want to be able to choose a file.
For this I use a placeholder.
Later when executing a command and filling the placeholder, I would like to have the same ability to show files based on the bash completion settings.
Is there some plans to include such feature?

**Comment by RamiAwar** [comment]
Hey @eeezyy ! Yes we have plans to include this as it's requested a lot. We're looking into how we could support this :) Just a bit challenging, but will probably be the next big feature worked on.
